code/go/0chain.net/core/common/errors_test.go

```diff
@@ -57,15 +57,15 @@ func TestInvalidRequest(t *testing.T) {
 		{
 			name: "",
 			args: args{msg: "msg"},
-			want: NewError("invalid_request", fmt.Sprintf("Invalid request (%v)", "msg")),
+			want: errors.New("invalid_request", fmt.Sprintf("Invalid request (%v)", "msg")),
```

Comment:
it can't be replaced with new errors package. because it is testing common/errors package.

Reply:
BTW: In this PR, all of references to common/errors are replaced with 0chain/errors package. is it better to delete common/errors.go and common/errors_test.go?

Reply:
> it can't be replaced with new errors package. because it is testing common/errors package.

I have removed the part of code in `common/errors.go` which is redundant with the `0chain/errors` package. 
Now, `common/errors.go` only has custom methods that are specific to this codebase. 

https://github.com/0chain/0chain/blob/089f1f26d4350c58f41c2372fea0caba064f52ee/code/go/0chain.net/core/common/errors.go#L32

> BTW: In this PR, all of references to common/errors are replaced with 0chain/errors package. is it better to delete common/errors.go and common/errors_test.go?

No, since `common/errors.go` contains custom methods that are specific to this codebase.